From a824eebef4854f1ba7b0e1a40c1c825af9a7a1f4 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Thu, 2 Apr 2020 01:27:46 +0200
Subject: [PATCH] sound: Fix translation of "System Sounds"

The wrong package name got passed for the gvc subproject, so an empty
gettext package name ended up being used to look for the "System Sounds"
translation.

Pass the package name so that the translation works as expected.

Closes: #426
---
 meson.build | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/meson.build b/meson.build
index 9eabd0f15..b563d214e 100644
--- a/meson.build
+++ b/meson.build
@@ -93,6 +93,8 @@ libgvc = subproject(
   default_options: [
     'static=true',
     'alsa=false',
+    'package_name=' + control_center_gettext,
+    'package_version=' + meson.project_version(),
   ]
 )
 libgvc_dep = libgvc.get_variable('libgvc_dep')
-- 
2.25.1

